packages/package-json/src/index.js

```diff
@@ -1,358 +1,252 @@
+	 *
+	 * @param {string} field
+	 * @returns {*}
+	 */
+	function getField(field) {
+		return workingContents[field];
+	}
+
+	/**
+	 * Set the value for a specific field in the `package.json` object.
+	 *
+	 * @param {string} field
+	 * @param {*} value
+	 *
+	 * @returns {object} - changelog entry
+	 */
+	function setField(field, value) {
+		const changes = { event: "setField", field };
+
+		const fieldAlreadyExists = typeof workingContents[field] !== "undefined";
+
+		if (fieldAlreadyExists) {
+			changes.previousValue = workingContents[field];
+		}
+
+		workingContents[field] = value;
+
+		return changelog.createEntry(changes);
+	}
+
+	/**
+	 * Require a package to exist as a dependency in `package.json`.
+	 *
+	 * @param {object} options
+	 * @param {string} options.pkg - Package name
+	 * @param {string} options.version
+	 * @param {string} options.field
+	 *
+	 * @returns {object} - changelog entry
+	 */
+	function requireDependency({ pkg, version, field }) {
+		if (!dependencyFields.includes(field)) {
+			throw new Error(
+				`package-json#requireDependency: Invalid field specified '${field}'. Valid fields: ${dependencyFields.join(
+					", "
+				)}`
+			);
+		}
+
+		if (!workingContents[field]) {
+			workingContents[field] = {};
+		}
+
+		const dependencies = workingContents[field];
+
+		const changes = { event: "requireDependency", field, pkg, version };
+
+		const dependencyAlreadyExists = typeof dependencies[pkg] !== "undefined";
+		changes.alreadyExisted = dependencyAlreadyExists;
+		if (dependencyAlreadyExists) {
+			changes.previousValue = dependencies[pkg];
+		}
+
+		dependencies[pkg] = version;
+
+		return changelog.createEntry(changes);
+	}
+
+	/**
+	 * Remove a package as a dependency from `package.json`.
+	 *
+	 * @param {object} options
+	 * @param {string} options.pkg
+	 * @param {string} options.field
+	 *
+	 * @returns {object|boolean} - changelog entry or `false` if dependency doesn't exist
+	 */
+	function removeDependency({ pkg, field }) {
+		if (!dependencyFields.includes(field)) {
+			throw new Error(
+				`package-json#removeDependency: Invalid field specified '${field}'. Valid fields: ${dependencyFields.join(
+					", "
+				)}`
+			);
+		}
+
+		if (!workingContents[field]) {
+			return false;
+		}
+
+		const dependencies = workingContents[field];
+
+		const changes = { event: "removeDependency", pkg, field };
+
+		if (typeof dependencies[pkg] !== "undefined") {
+			changes.previousValue = dependencies[pkg];
+			delete dependencies[pkg];
+		} else {
+			return false;
+		}
+
+		return changelog.createEntry(changes);
+	}
+
+	/**
+	 * Require a script to exist in the `scripts` field of `package.json`.
+	 *
+	 * @see https://docs.npmjs.com/misc/scripts
+	 *
+	 * @param {object} options
+	 * @param {string} options.lifecycleEvent - e.g. start, test, build, deploy
+	 * @param {string} options.command
+	 *
+	 * @returns {object} - changelog entry
+	 */
+	function requireScript({ lifecycleEvent, command }) {
```

Comment:
```suggestion
	function requireScript({ stage, command }) {
```

From https://docs.npmjs.com/misc/scripts, "additionally, arbitrary scripts can be executed by running npm run-script <stage>", I think `stage` is less prescriptive than `lifecycleEvent` and more in line with the documentation.

Reply:
I agree, `stage` is less prescriptive. This is something that @jenniferemshepherd and I debated renaming, but we couldn't come up with a good reason to change the name to `stage`. Your quote from the npm documentation gives us a great reason to change it :+1:

Reply:
Changed.